tcp/mux.go

```diff
@@ -79,6 +79,7 @@ func (mux *Mux) Serve(ln net.Listener) error {
 			continue
 		}
 		if err != nil {
+			mux.Logger.Printf("tcp.Mux.Serve: call to Accept() at %s failed, closing all listeners - %s", ln.Addr(), err)
```

Comment:
Is it possible to set a level on these log messages? i.e. error or warning?

Reply:
Most logging in InfluxDB is through the `zap` logger but `Mux` hasn't been updated to use that. We'd either have to update it to use `zap` or simulate the same structured logging format.

@davidby-influx since one intended audience of the log is users that may not read code, we generally avoid logging code. Can you convert these messages to something unique enough to allow engineers to located the source but also more friendly to users / non-coders?

Reply:
Rewrote log messages to match formatting of other message from tcp.Mux, made more user-friendly.

Reply:
In general you may want to consider using the `%v` verb for printing error messages; not `%s`.